src/tsconfig.ts

```diff
@@ -163,11 +163,23 @@ export function readFile (filename: string): Promise<any> {
         return reject(err)
       }
 
+      let obj
       try {
-        return resolve(parse(contents, filename))
```

Comment:
You'd need to keep the signature backward compatible here, otherwise it's a breaking change. I'd just make it optional, it used to be used for debugging and would still be useful to figure out which file has the syntax error.